dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -478,11 +478,14 @@ boolean hasValidSuperclass(JavaClass childJavaClass, JavaClass parentJavaClass)
     return constantPoolIndicesForTarget.build();
   }
 
+  private static final ImmutableSet<String> ERRORS_CAUGHT_IN_SOURCE =
+      ImmutableSet.of(NoClassDefFoundError.class.getName(), LinkageError.class.getName());
+
   /**
-   * Returns true if {@code sourceClassName} has a method that has an exception
-   * handler for {@link NoClassDefFoundError}.
+   * Returns true if {@code sourceClassName} has a method that has an exception handler for {@link
+   * NoClassDefFoundError} or {@link LinkageError}.
    */
-  boolean catchesNoClassDefFoundError(String sourceClassName) {
+  boolean catchesLinkageError(String sourceClassName) {
```

Comment:
NoClassDefFoundError is a subclass of LinkageError. So updating the function name.